velox/docs/functions/spark/conversion.rst

```diff
@@ -24,6 +24,23 @@ Valid examples:
   SELECT cast(1234 as tinyint); -- -46
   SELECT cast(1234567 as smallint); -- -10617
 
+From interval year month
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Casting from interval year month to integer returns the total number of months in the interval.
+The conversion is straightforward where 1 year = 12 months.
+
+Valid examples:
+
+::
+
+  SELECT cast(INTERVAL '1' YEAR as integer); -- 12
```

Comment:
I got the 1 for the sql in Spark. Is the query correct for the cast from `interval year month` to `integer`?

> spark.sql("SELECT cast(INTERVAL '1' YEAR as integer);").show(false)
+------------------------------+
|CAST(INTERVAL '1' YEAR AS INT)|
+------------------------------+
|1                             |
+------------------------------+

Reply:
The update seems not be pushed in. Could you confirm?